qa-niaid.planx-pla.net/manifest.json

```diff
@@ -18,7 +18,7 @@
     "revproxy": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/nginx:2022.05",
     "sheepdog": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/sheepdog:2022.05",
     "sower": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/sower:2022.05",
-    "portal": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/data-portal:feat_studyViewerIdpFlag",
+    "portal": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/data-portal:master",
```

Comment:
```suggestion
    "portal": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/data-portal:2022.05",
```

Reply:
would rather set it to master so we can see the latest changes, or does that release need to be tested?

Reply:
oh i see, then let's create a new data-portal release and deploy it?

Reply:
ok ill do that after i merge my latest changes